### PR TITLE
Plugwise Stretch changes for #40108

### PR DESCRIPTION
**Pull request by CoMPaTech**
## Proposed change
Add Stretch product to listed and supported devices. Add indication of using `stretch` as a username.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/40108
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by CoMPaTech** [comment]
Sorry incorrect link to parent PR first time and too soon on the close/repon for probot. Should be fine now

File: source/_integrations/plugwise.markdown [hunk 1]
```diff
@@ -16,16 +16,16 @@ ha_domain: plugwise
 ---
 
 This enables [Plugwise](https://www.plugwise.com) components with a central Smile gateway to be integrated. This integration talks locally to your **Smile** interface, and you will need its password and IP address.
-The platform supports [Anna](https://www.plugwise.com/en_US/products/anna), [Adam (zonecontrol)](https://www.plugwise.com/en_US/zonecontrol) and [P1](https://www.plugwise.com/en_US/products/smile-p1) Smile products. See below list for more details.
+The platform supports [Anna](https://www.plugwise.com/en_US/products/anna), [Adam (zonecontrol)](https://www.plugwise.com/en_US/zonecontrol), [P1](https://www.plugwise.com/en_US/products/smile-p1) Smile products and their [Stretch](https://www.plugwise.com/nl_NL/het-systeem) product. See below list for more details.
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: their [Stretch] --> the [Stretch]

File: source/_integrations/plugwise.markdown [hunk 2]
```diff
@@ -151,3 +151,9 @@ Smile P1 (DSMR):
  - v4.0
  - v3.3
  - v2.5
+ - v2.1
+
+Stretch (power switches):
+
+ - v3.1
+ - v2.3
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: Maybe we should write here v1.x, 2.x, etc. as we now only check the major version-number in Plugwise-Smile.

File: source/_integrations/plugwise.markdown [hunk 3]
```diff
@@ -142,18 +142,18 @@ Adam (zone_control):
...
 Smile P1 (DSMR):
 
- - v4.0
- - v3.3
- - v2.5
- - v2.1
+ - v4.x
+ - v3.x
+ - v2.x
+ - v2.x
```
Comment (bouwew) [inline source/_integrations/plugwise.markdown]: Please remove the double 2.x

**Review by bouwew — APPROVED** [review]
All looking good!

**Review by frenck — APPROVED** [review]

**Pull request merged**